src/models/data/cipherData.ts

```diff
@@ -43,6 +44,7 @@ export class CipherData {
         this.folderId = response.folderId;
         this.userId = userId;
         this.edit = response.edit;
+        this.viewPassword = response.viewPassword;
```

Comment:
Assuming this will need to change once changes are made to https://github.com/bitwarden/server/pull/743 per https://github.com/bitwarden/server/pull/743#discussion_r428747641